packages/sui-lint/package.json

```diff
@@ -38,7 +38,7 @@
     "prettier": "2.7.1",
     "stylelint": "14.11.0",
     "stylelint-config-prettier": "9.0.3",
-    "stylelint-config-recommended-scss": "7.0.0",
+    "stylelint-config-recommended-scss": "9.0.1",
```

Comment:
anything relevant on the changelog from major 7 to 8 and from major 8 to 9?

Reply:
Checking it seems like the correct one to use is the 8 let me check if it works
<img width="611" alt="image" src="https://github.com/SUI-Components/sui/assets/21960669/bfdbb723-37cf-4795-a675-f0d599b8384f">

Reply:
It works, so I will downgrade it to 8 since we have the styleling-config at version 9 witch is the recommended one for the version 8